tests/fuzzer/requirements.txt

```
@@ -0,0 +1,5 @@
+bcrypt@3.2.2
+cffi@1.15.0
+openssh-key-parser@0.0.2
```

Did you have a reason to pin the library under test to 0.0.2 rather than 0.0.5?
```suggestion
openssh-key-parser@0.0.5
```

---

I had a reason during research but not that I think about it, since this file is now included in openssh-key-parser, the inclusion of any version of the package in this requirements file is probably not useful. I would just delete the line entirely.